### PR TITLE
Fix/Typo in BQ table name

### PR DESCRIPTION
**Pull request by jungeunyoon**

### CODE REVIEW

**Comment by jungeunyoon** [comment]
[[TESTING] - Twitter Component BQ logs not logging (1)](https://trello.com/c/EJ3X3MZp/4858-testing-twitter-component-bq-logs-not-logging-1)

File: src/rise-twitter.js [hunk 1]
```diff
@@ -19,12 +19,15 @@ export default class RiseTwitter extends HTMLElement {
     this.isPreview = false;
```
Comment (jungeunyoon) [inline src/rise-twitter.js]: Most of these changes are from previous PRs. Not sure why they're being registered as differences.

**Comment by jungeunyoon** [comment]
@rodrigopavezi Please re-review additions, thanks!

File: src/tweet.js [hunk 2]
```diff
@@ -33,6 +33,8 @@ export default class Tweet {
           this.getTransition().start();
         })
         .catch((error) => {
+          console.log('error displaying tweets', error);
+          if (this.logger) {this.logger.error(`Invalid Tweets`);}
```
Comment (rodrigopavezi) [inline src/tweet.js]: @jungeunyoon I think you don't need to check logger anymore here right based on your changes to the logger that checks if LMS is connected. Cheers

Comment (jungeunyoon) [reply src/tweet.js]: @rodrigopavezi Isn't the purpose of checking if logger exists for preview mode where logger is null? Referencing a function `(error)` of an undefined class is problematic.

Comment (rodrigopavezi) [reply src/tweet.js]: @jungeunyoon I see, sorry.

Comment (jungeunyoon) [reply src/tweet.js]: @rodrigopavezi no worried! good to point out anything you see 👍

**Pull request merged**